index.html

```diff
@@ -4330,7 +4330,7 @@ <h2>Canonicalization</h2>
   </p><p>
   Some of the choices leading to the efficient computation of a canonical form, 
   such as the suppression of white space and sorting of elements in objects 
-  do make the representation less human readable.
+  do make the representation less human readable, more verbose, and more redundant.
```

Comment:
This addition looks contradictory to me. Why should sorting and/or suppressing white-spaces make it more verbose and redundant?

Reply:
I will try rewording that.  Suppression of whitespace does not make it more verbose, but other choices may.   Or I may just delete that addition.

Reply:
decided to just take those extra words out.